### ISSUE TITLE
hCard formats?

### ISSUE THREAD

**Issue by jpdevries** [Thinkful-Ed/react-address-book-solution#2]
Might be fun to demonstrate hCard formats but doing so would require changing the JSON structure
http://microformats.org/wiki/hcard

**Comment by oampo** [comment]
Yeah, that would be nice, but I think it's probably beyond the scope of these drills - the aim for the students is to get some initial experience with React components, and I'd rather not get that mixed in with a discussion of microformats.
